src/Expectation.php

```diff
@@ -394,6 +394,24 @@ public function toContain(...$needles): Expectation
         return $this;
     }
 
+    /**
+     * Asserts that $needle is an element of the value ignoring case.
+     *
+     * @param mixed $needles
+     */
+    public function toContainIgnoringCase(...$needles): Expectation
+    {
+        foreach ($needles as $needle) {
+            if (is_string($this->value)) {
+                Assert::assertStringContainsStringIgnoringCase($needle, $this->value);
+            } else {
+                Assert::assertContains($needle, $this->value);
```

Comment:
Feels weird to me that - this assert contains gets applied with any logic to ignore the case.